x/posts/internal/types/msgs.go

```diff
@@ -86,7 +86,7 @@ func (msg MsgEditPost) Type() string { return ActionEditPost }
 // ValidateBasic runs stateless checks on the message
 func (msg MsgEditPost) ValidateBasic() sdk.Error {
 	if !msg.PostID.Valid() {
-		return sdk.ErrUnknownRequest("Invalid post id")
+		return sdk.ErrUnknownRequest("Invalid post newID")
```

Comment:
I think this is a refactoring error

```suggestion
		return sdk.ErrUnknownRequest("Invalid post id")
```